control-plane/rest/service/src/v0/jsongrpc.rs

```diff
@@ -22,12 +22,29 @@ impl apis::actix_server::JsonGrpc for RestApi {
         Path((node, method)): Path<(String, String)>,
         Body(body): Body<Value>,
     ) -> Result<Value, RestError<RestJsonError>> {
-        let result = MessageBus::json_grpc_call(JsonGrpcRequest {
-            node: node.into(),
-            method: method.into(),
-            params: body.to_string().into(),
-        })
-        .await?;
+        let client = match json_grpc() {
```

Comment:
hmm, what about json_grpc returning a Result instead (with this error in case of not configured), and so here we'd just to:
```rust
json_grpc()?.call(...)...
```